docs/pages/enroll-resources/machine-id/workload-identity/federation.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: SPIFFE Federation
+description: An overview of the Teleport Workload Identity SPIFFE Federation feature.
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. This means that some
+features may be missing. We're actively looking for design partners to help us
+shape the future of Workload Identity and would love to
+[hear your feedback](mailto:product@goteleport.com).
+</Admonition>
+
+Federation is the ability for a trust relationship to be established between
+two trust domains, allowing workloads in one trust domain to validate the
+identities of workloads in the other trust domain.
+
+SPIFFE sets out a standard for federation that allows trust domains managed by
+different implementations to federate with one another. For example, a trust
+domain managed by Teleport Workload Identity could federate with a trust domain
+managed by SPIRE.
+
+The SPIFFE Federation process relies on the exchange of trust bundles between
+the trust domains. These trust bundles contain the certificates and public keys
+of the trust domain's issuer.
+
+Federation relationships are "one way" meaning that workloads within one trust
+domain can validate the identities of workloads in the other trust domain, but,
+workloads in the other trust domain cannot validate the identities of workloads
+in the first trust domain. It is typical to therefore establish the federation
+relationship in both directions.
+
+<Admonition type="warning" title="Teleport Enterprise Required">
+A valid Teleport Enterprise license is required to use the federation features
+of Teleport Workload Identity.
+</Admonition>
+
+## Federation to Teleport Workload Identity
+
+This section discusses how to configure another trust domain to trust a Teleport
+Workload Identity hosted trust domain.
+
+In order to configure a trust domain to trust Teleport Workload Identity, the
+control plane of the trust domain must be able to access the SPIFFE Bundle
+Endpoint exposed on the Teleport Proxy.
+
+The Teleport Workload Identity SPIFFE Bundle Endpoint implements the "https_web"
+profile as defined by the SPIFFE Federation specification. This means that the
+endpoint is served using the standard web TLS certificates configured for your
+Teleport Proxy.
+
+The SPIFFE Bundle Endpoint is exposed on the Teleport Proxy at the path
+`/webapi/spiffe/bundle.json`.
+
+## Federation from Teleport Workload Identity
+
+This section discusses how to configure a Teleport Workload Identity hosted
+trust domain to trust another trust domain.
+
+Teleport Workload Identity only supports federating with trust domains that
+offer a SPIFFE Bundle Endpoint that abides by the "https_web" profile as defined
+by the SPIFFE Federation specification.
+
+To configure Teleport Workload Identity to trust another trust domain, the
+`spiffe_federation` resource is used.
```

Comment:
I think the passive voice makes this paragraph less clear. What uses the `spiffe_federation` resource? How does it use the resource?

Reply:
7a9be891d5